sdk/openai/Azure.AI.OpenAI/src/Custom/FineTuning/AzureFineTuningClient.Extensions.cs

```diff
@@ -0,0 +1,100 @@
+    /// <param name="jobId">The identifier for the fine tuning job to delete.</param>
+    /// <param name="options">(Optional) The request options to use.</param>
+    /// <returns>True if the job deletion request was sent successfully, false otherwise.</returns>
+    /// <remarks>The Azure OpenAI service will always return a success (HTTP 204) regardless of whether or not
+    /// the job you are trying to delete exists.</remarks>
+    [Experimental("AOAI001")]
+    public static bool DeleteJob(this FineTuningClient client, string jobId, RequestOptions? options = null)
+#pragma warning disable AZC0106 // Non-public asynchronous method needs 'async' parameter.
+        => DeleteJobSyncOrAsync(client, false, jobId, options).EnsureCompleted();
+#pragma warning restore AZC0106 // Non-public asynchronous method needs 'async' parameter.
+
+    /// <summary>
+    /// Deletes an Azure OpenAI fine tuning job.
+    /// </summary>
+    /// <param name="client">The Azure OpenAI fine tuning client.</param>
+    /// <param name="jobId">The identifier for the fine tuning job to delete.</param>
+    /// <param name="options">(Optional) The request options to use.</param>
+    /// <returns>True if the job deletion request was sent successfully, false otherwise.</returns>
+    /// <remarks>The Azure OpenAI service will always return a success (HTTP 204) regardless of whether or not
+    /// the job you are trying to delete exists.</remarks>
+    [Experimental("AOAI001")]
+    public static Task<bool> DeleteJobAsync(this FineTuningClient client, string jobId, RequestOptions? options = null)
+        => DeleteJobSyncOrAsync(client, true, jobId, options).AsTask();
+
+    /// <summary>
+    /// Deletes an Azure OpenAI fine tuning job.
+    /// </summary>
+    /// <param name="client">The Azure OpenAI fine tuning client.</param>
+    /// <param name="isAsync">True if this should be an async network request, false otherwise/</param>
+    /// <param name="jobId">The identifier for the fine tuning job to delete.</param>
+    /// <param name="options">(Optional) The request options to use.</param>
+    /// <returns>True if the job deletion request was sent successfully, false otherwise.</returns>
+    /// <remarks>The Azure OpenAI service will always return a success (HTTP 204) regardless of whether or not
+    /// the job you are trying to delete exists.</remarks>
+    [Experimental("AOAI001")]
+    internal static async ValueTask<bool> DeleteJobSyncOrAsync(this FineTuningClient client, bool isAsync, string jobId, RequestOptions? options = null)
+    {
+        Argument.AssertNotNull(client, nameof(client));
+        Argument.AssertNotNullOrEmpty(jobId, nameof(jobId));
+
+        var azureClient = client as AzureFineTuningClient;
+        if (azureClient == null)
+        {
+            throw new InvalidOperationException("Only supported on Azure OpenAI fine tuning clients");
+        }
+
+        PipelineMessage message = azureClient.Pipeline.CreateMessage();
```

Comment:
Despite being additive, let's move the protocol-level concepts (message creation, etc) into the protocol class -- if for nothing else than to keep it centralized for maintenance.

Reply:
I will add an internal only protocol method for this,